### ISSUE TITLE
 Modal - focus ring around 'Close' button is not visible when user opens modal

### ISSUE THREAD

**Issue by mihalichzh** [semrush/intergalactic#711]
**Describe the bug**
Noticed, that the focus ring is not initially visible when a user opens modal.

**To Reproduce**
1. Open modal (first example from [docs](https://developer.semrush.com/intergalactic/components/modal/modal-code/) can be used)
2. Check focus ring around 'Close' button

**Expected behavior**
Focus ring is visible

**Actual behavior**
Focus ring is not visible (see att. screencast)

https://github.com/semrush/intergalactic/assets/39348763/6067bb48-6310-4b20-a04f-1f0623ac432e

**Comment by msereniti** [comment]
Hi, the issue was just fixed!
https://github.com/semrush/intergalactic/releases/tag/v15.10.0

Also need to note that focus ring will be visible only if modal is being opened by keyboard interaction
